e2e/cephfs.go

```diff
@@ -118,6 +118,34 @@ func validateSubvolumeCount(f *framework.Framework, count int, fileSystemName, s
 	}
 }
 
+func validateSnapshotCount(f *framework.Framework, count int, subvolumegroup string, pvc *v1.PersistentVolumeClaim) {
+	pvc, err := f.ClientSet.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(
+		context.TODO(),
+		pvc.Name,
+		metav1.GetOptions{},
+	)
+	if err != nil {
+		e2elog.Failf("failed to get pvc: %v", err)
+	}
+	pv, err := f.ClientSet.CoreV1().PersistentVolumes().Get(
+		context.TODO(),
+		pvc.Spec.VolumeName,
+		metav1.GetOptions{},
+	)
+	if err != nil {
+		e2elog.Failf("failed to get pv: %v", err)
+	}
+	subVolumeName := pv.Spec.CSI.VolumeAttributes["subvolumeName"]
+	fsName := pv.Spec.CSI.VolumeAttributes["fsName"]
+	snaps, err := listCephFSSnapshots(f, fsName, subVolumeName, subvolumegroup)
+	if err != nil {
+		e2elog.Failf("failed to list subvolume snapshots: %v", err)
+	}
+	if len(snaps) != count {
+		e2elog.Failf("snapshots [%v]. snapshots count %d not matching expected count %v", snaps, len(snaps), count)
```

Comment:
**nit:** weirdly mixing `%d` and `%v` in the failure string. Both arguments are `int`, so use `%d` for both too?